lua/nvim-tree.lua

```diff
@@ -827,6 +859,22 @@ function M.setup(conf)
     log.raw("config", "%s\n", vim.inspect(opts))
   end
 
+  if M.config.renderer.size.column_width < 6 then
+    notify.warn "`size.right_padding` is a small number, problably won't show any size numbers, try using 10."
```

Comment:
nvim-tree is a sharp tool, users can cut themselves as they wish.

Whenever we try to do something like this users complain vehemently.

Reply:
Glad to know it